frontend/pages/features.tsx

```diff
@@ -24,16 +25,28 @@ export default function Page() {
 
 	return (
 		<main className="flex flex-1">
-			<Head>
-				<title>Kioku</title>
-				<meta name="description" content="Kioku" />
-				<link rel="icon" href="/favicon.ico" />
-				<link rel="alternate" hrefLang="en" href="https://app.kioku.dev/features" />
-				<link rel="alternate" hrefLang="de" href="https://app.kioku.dev/de/features" />
-			</Head>
+			<NextSeo
+				title={_(msg`Kioku | Features of the Kioku platform`)}
+				description={_(
+					msg`Explore the features of the Kioku platform! The most important features are collaborative flashcard creation and compability with anki`
```

Comment:
features twice sounds weird
```suggestion
					msg`Explore the features of the Kioku platform! The most important features are collaborative flashcard creation and compability with Anki`
```